routingpy/routers/osrm.py

```diff
@@ -89,6 +89,7 @@ def __init__(
 
     def directions(
         self,
+        profile,
```

Comment:
I re-added these and changed the other functions' signature as well to be consistent with valhalla's function signatures. even if `profile` is not used anywhere